assets/js/modules/reader-revenue-manager/components/settings/SettingsEdit.test.js

```diff
@@ -393,4 +403,82 @@ describe( 'SettingsEdit', () => {
 			).not.toBeInTheDocument();
 		} );
 	} );
+
+	describe( 'GA event tracking', () => {
+		afterEach( () => {
+			mockTrackEvent.mockClear();
+		} );
+
+		it( 'should track an event if snippet mode was changed', async () => {
+			fetchMock.postOnce( settingsEndpoint, () => ( {
+				body: {
+					...settings,
+					snippetMode: 'per_post',
+				},
+				status: 200,
+			} ) );
```

Comment:
The anonymous function is unnecessary here:

```suggestion
			fetchMock.postOnce( settingsEndpoint, {
				body: {
					...settings,
					snippetMode: 'per_post',
				},
				status: 200,
			} );
```

Reply:
Updated, thanks Tom!